platform-api/src/main/resources/application.yml

```diff
@@ -190,6 +190,7 @@ datacater:
     kafka:
       devservices:
         port: 9092
+        provider: kafka-native
```

Comment:
I guess this change means we expilictly can't support redpanda brokers. How do we feel about this?

Reply:
We already took this step for the [pipeline implementation](https://github.com/DataCater/datacater/commit/6b5bc2bb5ecf2408d3eb7eb15bed43acd6d1f1a9). Thus, I don't have any hard feelings about it.